apps/project/migrations/0063_auto_20200715_1131.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.15 on 2020-07-15 11:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('project', '0062_auto_20200715_1015'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='project',
+            name='organizations',
+            field=models.ManyToManyField(blank=True, through='project.ProjectOrganization', to='organization.Organization'),
+        ),
```

Comment:
Please create a single migration for these two.
Steps:
- revert back to migration 0061 [ before org field was added] `python manage.py migrate project 0061`
- remove the migration files
- makemigration and migrate again.

**NOTE**: if you have switched branches since these migrations and ran migrations on other branches, there might be some migration issues.

Reply:
cc: @AdityaKhatri @tnagorra @subinasr